src/conf.c

```diff
@@ -1531,6 +1531,8 @@ static void string_print(char *str, int id, snd_output_t *out)
 	case '.':
 	case '{':
 	case '}':
+	case '[':
+	case ']':
```

Comment:
The change looks file. Add 'Signed-off-by:' line to your commit.

Reply:
I've added the comment to the commit (through a git force-push, is this possible through a normal push?)